src/discussions/comments/CommentsView.jsx

```diff
@@ -41,12 +45,19 @@ function CommentsView({ intl }) {
   return (
     <div className="discussion-comments d-flex flex-column w-100 ml-3">
       <div className="mb-2">
+        { thread.closed && BANNER_TYPE.closed(intl.formatMessage(bannerMessages.closed)) }
+        { thread.abuseFlagged && BANNER_TYPE.reported(intl.formatMessage(bannerMessages.contentReported)) }
+        { thread.pinned && BANNER_TYPE.pinned(intl.formatMessage(bannerMessages.pinned)) }
         <div className="list-group list-group-flush">
           <Post post={thread} />
           <div className="list-group">
             {comments.map(comment => (
-              <div key={comment.id} className="list-group-item list-group-item-action">
-                <Comment comment={comment} />
+              <div key={comment.id}>
+                { comment.endorsed && BANNER_TYPE.endorsed(intl.formatMessage(bannerMessages.endorsed)) }
+                { comment.abuseFlagged && BANNER_TYPE.reported(intl.formatMessage(bannerMessages.contentReported)) }
```

Comment:
We can just move this inside the Comment component? Then they will show up for both comments here and replies to comments. 
Currently, the banner doesn't show for replies.